skema/program_analysis/CAST/matlab/matlab_tree_builder.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os.path
+import pprint
+from pathlib import Path
+from typing import Any, Dict, List, Union
+from tree_sitter import Language, Parser, Node, Tree
+
+from skema.program_analysis.tree_sitter_parsers.build_parsers import INSTALLED_LANGUAGES_FILEPATH
+
+class MATLAB_TREE_BUILDER(object):
+    def __init__(self):
+        
+        # Create the tree-sitter MATLAB parser
+        self.parser = Parser()
+        self.parser.set_language(
+            Language(
+                Path(Path(__file__).parent, INSTALLED_LANGUAGES_FILEPATH),
```

Comment:
This can now be simplified to just `INSTALLED_LANGUAGES_FILEPATH`

Reply:
@vincentraymond-ua , to clarify, are you saying that `Path(Path(__file__).parent, INSTALLED_LANGUAGES_FILEPATH)` can be replaced with just `INSTALLED_LANGUAGES_FILEPATH`?

Reply:
@jastier , looking at the definition of `INSTALLED_LANGUAGES_FILEPATH`, you should be able to replace `Path(Path(__file__).parent, INSTALLED_LANGUAGES_FILEPATH)` with just `INSTALLED_LANGUAGES_FILEPATH`

Reply:
Done.